index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="css/my_portfolio.css">
+</head>
+<body>
+    <header id="header" class="header"> 
+        <div class="logo">John Doe</div>
```

Comment:
- [ ] Hi, Please use the correct font and font-weight for this logo text. The correct font should be Roboto not Poppins. 
## Your Design
![image](https://user-images.githubusercontent.com/18730106/143315127-3e67dd0b-69ff-41fd-8c95-4af10a36d6d7.png)

## Figma Design
![image](https://user-images.githubusercontent.com/18730106/143315088-ed6f6a49-27eb-45ba-a17a-df45f70222ca.png)